### ISSUE TITLE
Mod Suggestions

### ISSUE THREAD

**Issue by carsonharrell** [snipergaming888/snipergaming888.github.io#8]
VS Bob
VS Tree
Mid Fight Masses

If i think of more i'll reply to this

**Comment by hdboye** [comment]
i made a pull request for vs bob so that one should be added soon

**Comment by Zadgardo** [comment]
Heyo i followed you over here from itch io and what happened to your account (itch io)? and can you add the K.O.U mod. Thanks for porting this!

**Comment by snipergaming888** [comment]
> Heyo i followed you over here from itch io and what happened to your account (itch io)? and can you add the K.O.U mod. Thanks for porting this!

nope! the Kou mod creator does not want his mod to be redistributed, for he/she will (hopefully) be adding browser support when he/she (if) ports it to kade engine.

**Comment by snipergaming888** [comment]
also this is gonna stay open so request as much as you want

**Comment by Zadgardo** [comment]
Oh alright it was such a good mod too T^T

**Comment by Zadgardo** [comment]
and is there any anime mods? You don't have to go digging for some I just wanna know if you know any (besides Boyfriends Circulation)

**Comment by carsonharrell** [comment]
If you can get permission to use them, these mods would be cool:

Lazybones [FULL WEEK]

**Comment by Zadgardo** [comment]
Sniper or anyone What does the naughtiness setting do on the official game when checked off. Does it increase or decrease it because I see no difference

**Comment by Zadgardo** [comment]
Can you please at the CJ mod it looks interesting

**Comment by Zadgardo** [comment]
> i made a pull request for vs bob so that one should be added soon

Whats a pull request?

**Comment by snipergaming888** [comment]
> Can you please at the CJ mod it looks interesting

Yeah eventually

**Comment by snipergaming888** [comment]
> > i made a pull request for vs bob so that one should be added soon
> 
> Whats a pull request?

When someone request a something to be added, if accepted, those files will be added to the repository.

**Comment by snipergaming888** [comment]
> Sniper or anyone What does the naughtiness setting do on the official game when checked off. Does it increase or decrease it because I see no difference

I have no idea lmao

**Comment by Zadgardo** [comment]
> > Sniper or anyone What does the naughtiness setting do on the official game when checked off. Does it increase or decrease it because I see no difference
> 
> I have no idea lmao

Lol its fine I saw it in a stream from Woops and they said it removes blue balls and swearing

**Comment by Zadgardo** [comment]
> > Can you please at the CJ mod it looks interesting
> 
> Yeah eventually

Thanks bro!

**Comment by vackyton** [comment]
stress ez
